scss/_tabs.scss

```diff
@@ -318,6 +318,30 @@ ion-tabs {
         transition: color .2s ease;
       }
     }
+   &:not(.tabs-icon-left):not(.tabs-icon-top){
```

Comment:
Is there a way we could fix the source of the active text moving down instead of having to monkeypatch it with this css rule?

The problem is this rule is impossible to override and not maintainable in the future.